pkg/steps/source.go

```diff
@@ -32,13 +32,13 @@ var (
 	JobSpecAnnotation = fmt.Sprintf("%s/%s", CiAnnotationPrefix, "job-spec")
 )
 
-func sourceDockerfile(fromTag api.PipelineImageStreamTagReference, refs Refs) string {
+func sourceDockerfile(fromTag api.PipelineImageStreamTagReference, job *JobSpec) string {
 	return fmt.Sprintf(`FROM %s:%s
 ENV GIT_COMMITTER_NAME=developer GIT_COMMITTER_EMAIL=developer@redhat.com
 ENV REPO_OWNER=%s REPO_NAME=%s PULL_REFS=%s
-RUN umask 0002 && /usr/bin/release-ci cloneref --src-root=/go
+RUN umask 0002 && /usr/bin/clonerefs --src-root=/go --log=-
```

Comment:
Ideally would import and use the JSON config from upstream here

Reply:
Instead of using JOB_SPEC?